public/index.html

```diff
@@ -28,6 +28,12 @@
       name="keyword"
       content="decentralized,videocall,audiocall,chat,milaap,India first fully decentralized video calling and chatting app,peer to peer, simple peer"
     />
+    <!--
```

Comment:
Add a TODO to remove it while pushing it in production with PWA

Also, create a issues about the same so that we don't forget it later.

Reply:
For preventing cache especially for mobile devices, will this work? (with changes to service worker)
[https://stackoverflow.com/questions/46080221/deployed-react-web-app-requires-hard-refresh](https://stackoverflow.com/questions/46080221/deployed-react-web-app-requires-hard-refresh)

Reply:
@naman-modi had the same thought that just unregistering service worker should work.
@veddandekar please check both the approaches.

I guess it's better if we go with unregistering service worker for now.